Modules/@babylonjs/react-native-windows/windows/include/XrContextHelperOpenXR.h

```diff
@@ -6,20 +6,24 @@
 
 namespace BabylonReactNative
 {
-    bool TryGetXrContext(facebook::jsi::Runtime& jsiRuntime, const facebook::jsi::Value& xrSession, IXrContextOpenXR*& xrContext)
+    bool TryGetXrContext(facebook::jsi::Runtime& jsiRuntime, IXrContextOpenXR*& xrContext)
     {
         xrContext = nullptr;
-        if (!xrSession.isObject() ||
-            !xrSession.asObject(jsiRuntime).hasProperty(jsiRuntime, "nativeXrContext") ||
-            !xrSession.asObject(jsiRuntime).hasProperty(jsiRuntime, "nativeXrContextType") ||
-            xrSession.asObject(jsiRuntime).getProperty(jsiRuntime, "nativeXrContextType").asString(jsiRuntime).utf8(jsiRuntime) != "OpenXR")
+        if (jsiRuntime.global().hasProperty(jsiRuntime, "navigator") &&
+            jsiRuntime.global().getProperty(jsiRuntime, "navigator").asObject(jsiRuntime).hasProperty(jsiRuntime, "xr"))
         {
-            return false;
+            auto nativeXr = jsiRuntime.global().getProperty(jsiRuntime, "navigator").asObject(jsiRuntime).getProperty(jsiRuntime, "xr").asObject(jsiRuntime);

```

Comment:
It would be nice if we didn't have to duplicate this quite so much. Like maybe if the pattern here was more like:
```c++
if (!jsiRuntime.global().hasProperty(jsiRuntime, "navigator")
{
    return false;
}

auto navigator{jsiRuntime.global().getProperty(jsiRuntime, "navigator").asObject(jsiRuntime)};
if (!navigator.hasProperty(jsiRuntime, "xr"))
{
    return false;
}

auto nativeXr{navigator.getProperty(jsiRuntime, "xr").asObject(jsiRuntime)};
...
```